using-mobile-extensions/adobe-target/README.md

```diff
@@ -116,9 +116,9 @@ Prefetched offer content does not persist across launches. The prefetch content
 #### Java
 
 
-#### Using the `TargetPrefetch` Constructor
+#### Using the `TargetPrefetch` constructor
 
-Using `TargetPrefetch` Constructor, you can create a `TargetPrefetch` instance with the specified data. It currently accepts location name and an optional `TargetParameters` object. The returned instance can be used with `prefetchContent`, which accepts a `TargetPrefetch` object list to prefetch offers for the specified mbox locations. 
+With `TargetPrefetch` constructor, you can create a `TargetPrefetch` instance with the specified data. It currently accepts the location name and an optional `TargetParameters` object. The returned instance can be used with `prefetchContent`, which accepts a `TargetPrefetch` object list to prefetch offers for the specified mbox locations.
```

Comment:
Please add "the" between "With" and `TargetPrefetch", so that it reads "With the `TargetPrefetch...".